modules/auxiliary/scanner/http/cgit_traversal.rb

```diff
@@ -0,0 +1,66 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Directory traversal in cgit',
```

Comment:
I would change the name to cgit directory traversal. More consistent with other modules and easier to find when you're looking for cgit